arangod/RestHandler/RestVocbaseBaseHandler.cpp

```diff
@@ -227,6 +228,15 @@ RestVocbaseBaseHandler::RestVocbaseBaseHandler(application_features::Application
 
 RestVocbaseBaseHandler::~RestVocbaseBaseHandler() = default;
 
+void RestVocbaseBaseHandler::prepareExecute(bool isContinue) {
+  //_dbNameContext = LogContext::current().push(LogContext::makeValue().with<logContextKeyDatabaseName>(_vocbase.name()));
```

Comment:
I think right now we only want to merge the infrastructure for context logging into devel, but we haven't defined (exactly) what values to log.
So I am fine with commenting this out. However, we should add a code comment here that explains the reason behind nothing being added here (likely "TODO: decide about which values to log in a future step").